cpu/esp32/Makefile.include

```diff
@@ -64,12 +66,18 @@ _ESP_WIFI_ANY_BASELIBS += -lphy -lstdc++
 # Baselibs needed when using esp_now module
 _ESP_NOW_BASELIBS = -lespnow -lmesh
 
-# NOTE: These checks can be turned into a normal conditional when #9913 is fixed
-BASELIBS += $(if $(filter esp_wifi_any,$(USEMODULE)),$(_ESP_WIFI_ANY_BASELIBS))
-BASELIBS += $(if $(filter esp_now,$(USEMODULE)),$(_ESP_NOW_BASELIBS))
+ifneq (,$(filter esp_wifi_any,$(USEMODULE)))
+  BASELIBS += $(_ESP_WIFI_ANY_BASELIBS)
```

Comment:
`_ESP_WIFI_ANY_BASELIBS` and `_ESP_NOW_BASELIBS` were only introduced for this work around and should be removed with this change.
```suggestion
  BASELIBS += -lcore -lrtc -lnet80211 -lpp -lcoexist -lwps -lwpa -lwpa2
  BASELIBS += -lphy -lstdc++
```